src/Microsoft.Azure.Functions.Extensions.Mcp/Http/SseRequestHandler.cs

```diff
@@ -134,9 +136,8 @@ public async Task HandleMessageRequest(HttpContext context, McpOptions mcpOption
         await context.Response.WriteAsync("Accepted");
 
         static Task WriteInvalidSessionResponse(string message, HttpContext httpContext)
-        {
-            return Results.BadRequest($"{message} Please connect to the /sse endpoint to initiate your session.").ExecuteAsync(httpContext);
-        }
+            => Results.BadRequest($"{message} Please connect to the /sse endpoint to initiate your session.")
```

Comment:
```suggestion
            => Results.BadRequest($"{message}. Please connect to the /sse endpoint to initiate your session.")
```

Reply:
Keeping the existing implementation with the calling code adding that period (already in place). The changes made to `SseRequestHandler` were primarily to deal with some of the changes made to common dependencies (granted, this one is an exception).

I'm applying the other method name changes though, as those are more impactful (even if out of scope).